docs/docs/frontend-protocol.md

```diff
@@ -117,6 +117,18 @@ multiple cursors and `chars` has the same number of lines as there are
 cursors, one line will be inserted at each cursor, in order; otherwise the full
 string will be inserted at each cursor.
 
+#### copy 
+
+`copy` -> Value
```

Comment:
I would have this whole line in the \`s, and I would have the return value be `String|Null`.